src/validation/nationality.validation.ts

```diff
@@ -176,3 +169,10 @@ const validateCommonNationality = (
   }
 }
 
+const invalidPattern = (input: string, regex: RegExp): boolean => {
+  const matchResult = input.match(regex);
+  if (matchResult == null || matchResult == undefined){
```

Comment:
return matchResult === null || matchResult === undefined;

Reply:
From Javascript docs - this may be enough:
return matchResult === null;

Reply:
or 
return input.test(regex);

Reply:
simplified to inline !input.match(regex)